python/stack_plots.py

```diff
@@ -409,7 +415,7 @@ def draw_plot(sampleDict, plotname, logY=True, logX=False, plotData=False, doRat
         lumi = testLumi
 
     yearenergy=""
-    if year!="all":
+    if year!="all" or lumi<100.0:
```

Comment:
What's the point of the ```or``` here? I understand correctly, this will print "13.8 fb^{-1} (all, 13 TeV)" for the partial unblinding case, while dropping the "all" may be nicer.

Reply:
It is to not just print `14` in this case. However, I agree.
Changed to `(2016-2018, 13 TeV)` in that case.